### ISSUE TITLE
PySide doesn't work in Python2

### ISSUE THREAD

**Issue by tfeldmann** [Homebrew/homebrew#44213]
I built PySide from source with "--with-python3". It works fine with python 3.5 but does not work in python 2.7.10. It seems it has problems with shiboken.

```
➜  ~  python3
Python 3.5.0 (default, Sep 14 2015, 02:37:27)
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.53)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from PySide import QtCore
>>>
➜  ~  python
Python 2.7.10 (default, Sep 14 2015, 02:26:06)
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.53)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from PySide import QtCore
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: dlopen(/usr/local/lib/python2.7/site-packages/PySide/QtCore.so, 2): Symbol not found: __Py_FalseStruct
  Referenced from: /usr/local/lib/libshiboken.cpython-35m-darwin.1.2.dylib
  Expected in: flat namespace
 in /usr/local/lib/libshiboken.cpython-35m-darwin.1.2.dylib
>>>
➜  ~  which python
/usr/local/bin/python
```

**Comment by tdsmith** [comment]
Can you share the output of `import sys; print(sys.path)` from python2?

**Comment by tfeldmann** [comment]
''
    '/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python27.zip'
    '/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7'
    '/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-darwin'
    '/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac'
    '/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac/lib-scriptpackages'
    '/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-tk'
    '/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-old'
    '/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload'
    '/Users/thomasfeldmann/Library/Python/2.7/lib/python/site-packages'
    '/usr/local/lib/python2.7/site-packages'
    '/usr/local/Cellar/matplotlib/1.4.3/libexec/lib/python2.7/site-packages'
    '/usr/local/Cellar/numpy/1.9.2_1/libexec/nose/lib/python2.7/site-packages'
    '/Library/Python/2.7/site-packages'

**Comment by tdsmith** [comment]
I'm testing a fix; it looks like we should be passing in a cmake variable called PYTHON_SUFFIX containing either "-python2.7" or a value constructed from the python SOABI configuration variable.

**Comment by tdsmith** [comment]
Should be fixed by 2c3be8d.

**Comment by tfeldmann** [comment]
Thanks, this fixes the problem.
